lib/src/bloc_builder.dart

```diff
@@ -1,9 +1,15 @@
 import 'package:bloc/bloc.dart';
 import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooked_bloc/hooked_bloc.dart';
 import 'package:hooked_bloc/src/injection/hook_injection_controller.dart';
 
 typedef BlocBuilderCondition<S> = bool Function(S current);
 
+/// Returns current state of BlocBase class
+///
+/// By default [useCubitBuilder] will not filter any states, unless specified [buildWhen] parameter or [builderCondition] in [HookedBloc.initialize]
+/// [buildWhen] local filter function, that will filter incoming states from [BlocBase.stream]
+///
 S useCubitBuilder<C extends BlocBase, S>(
```

Comment:
I would remove extra space  `///`
you can see here that they don't do any extra space between method and documentation. 
https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Reply:
right., done.